modules/nf-core/dragen/germline/meta.yml

```diff
@@ -0,0 +1,1493 @@
+---
+# yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/modules/meta-schema.json
+name: "dragen"
+description: The Illumina DRAGEN™ Bio-IT Platform is based on the highly reconfigurable DRAGEN Bio-IT Processor, which is integrated on a Field Programmable Gate Array (FPGA) card and is available in a preconfigured server that can be seamlessly integrated into bioinformatics workflows. The platform can be loaded with highly optimized algorithms for many different NGS secondary analysis pipelines.
+keywords:
+  - genomics
+  - variant calling
+  - germline
+tools:
+  - "dragen":
+      description: "The Illumina DRAGEN™ Bio-IT Platform is based on the highly reconfigurable DRAGEN Bio-IT Processor, which is integrated on a Field Programmable Gate Array (FPGA) card and is available in a preconfigured server that can be seamlessly integrated into bioinformatics workflows. The platform can be loaded with highly optimized algorithms for many different NGS secondary analysis pipelines."
+      homepage: "https://support-docs.illumina.com/SW/dragen_v42/Content/SW/FrontPages/DRAGEN.htm?searchText=explify-ref-db-dir"
+      documentation: "https://support-docs.illumina.com/SW/dragen_v42/Content/SW/FrontPages/DRAGEN.htm?searchText=explify-ref-db-dir"
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information
+        e.g. `[ id:'sample1', single_end:false , sex:'female']`;
+        options for sex: male, female, none, auto
+  - fastq:
+      type: file
+      description: FASTQ files to send to card (may be gzipped)
+      pattern: "*.{fastq,fq}{,.gz}"
+  - ora:
+      type: file
+      description: Specifies the input files for compression or decompression
+      pattern: "*.ora"
+  - bam:
+      type: file
+      description: Input BAM file to send to card
+      pattern: "*.bam"
+  - cram:
+      type: file
+      description: Input CRAM file to send to card
+      pattern: "*.cram"
```

Comment:
Could we make this one input and figure out based on extension which?

Reply:
do you want all sequence data containing inputs combined?

Reply:
I included more keywords. Let me know what you think. ☺️